app/views/layouts/_alerts.html.erb

```diff
@@ -0,0 +1,9 @@
+<% flash.each do |_, msg| %>
```

Comment:
change _ to some better name

Reply:
It's not being used anywhere in the block (generally rubocop prefers it this way when something is not being used)